meson_options.txt

```diff
@@ -269,7 +269,7 @@ option('dns-over-tls', type : 'combo', choices : ['auto', 'gnutls', 'openssl', '
        description : 'DNS-over-TLS support')
 option('dns-servers', type : 'string',
        description : 'space-separated list of default DNS servers',
-       value : '1.1.1.1 8.8.8.8 1.0.0.1 8.8.4.4 2606:4700:4700::1111 2001:4860:4860::8888 2606:4700:4700::1001 2001:4860:4860::8844')
+       value : '1.1.1.1#cloudflare-dns.com 8.8.8.8#dns.google 1.0.0.1#cloudflare-dns.com 8.8.4.4#dns.google 2606:4700:4700::1111#cloudflare-dns.com 2001:4860:4860::8888#dns.google 2606:4700:4700::1001#cloudflare-dns.com 2001:4860:4860::8844#dns.google')
```

Comment:
Hmmm
```console
$ host 1.1.1.1
1.1.1.1.in-addr.arpa domain name pointer one.one.one.one.
$ host cloudflare-dns.com
cloudflare-dns.com has address 104.16.249.249
cloudflare-dns.com has address 104.16.248.249
cloudflare-dns.com has IPv6 address 2606:4700::6810:f8f9
cloudflare-dns.com has IPv6 address 2606:4700::6810:f9f9
```

Reply:
Both cloudflare-dns.com and one.one.one.one work for me